playbooks/azure/openshift-cluster/example_params.yml

```diff
@@ -0,0 +1,34 @@
+---
+# need these to connect with a service principal
+# this didn't seem to work, had to resource to ~/.azure/credentials file
+# client_id: "{{ lookup('env','AZURE_CLIENT_ID') }}"
+# secret: "{{ lookup('env','AZURE_SECRET') }}"
+# subscription_id: "{{ lookup('env','AZURE_SUBSCRIPTION_ID') }}"
+# tenant: "{{ lookup('env','AZURE_TENANT') }}"
+
+# resource group parameters
+openshift_azure_create_resource_group: True
+openshift_azure_resource_group_name: openshifttestgroup
+openshift_azure_resource_location: eastus
+
+# storage account parameters
+openshift_azure_storage_account_type: Standard_LRS
+
+# virtual network parameters
+openshift_azure_virtual_network_address_prefixes: "10.10.0.0/16"
+openshift_azure_subnet_prefix: "10.10.0.0/24"
+
+# base vm parameters
+openshift_azure_vm_name: openshiftbasevm
+openshift_azure_vm_storage_container: "openshiftbasevm.vhd"
+openshift_azure_vm_size: Standard_D1
+openshift_azure_vm_admin_username: "openshift"
+openshift_azure_vm_admin_password: "OpenShift1!"
```

Comment:
What's preventing openshift_azure_vm_admin_password from being removed? Is it genuinely safe to include it as it is?

Reply:
We can probably remove this.  We use public ssh keys but it is possible to pass a password.

Reply:
We can use public ssh keys or passwords.  By default we use ssh public keys.  I will remove this example_params.yml file.